### ISSUE TITLE
Drone flight zone map integration

### ISSUE THREAD

**Issue by patranger** [osmandapp/OsmAnd#14424]
# 🚀 feature request

### Description

Sometimes, when hiking, you want to use the drones to check the terrain and take photos, and so I think it would be pretty cool if OsmAnd had a flight zone map integration or at least integration with the dronestagr.am website so that you could see photos of the area from the drone.

Such a function can even be useful when someone wants to go for a walk in peace where nothing will fly over his head.

**Comment by tradavyvy** [comment]
Drone (no-)fly zone integration would be verry nice. As a map feature like nature reserves or environmental zones, or as a map overlay, or even a seperate profile.
Integration with a (commercial) drone photo service might be a step to far.

**Comment by patranger** [comment]
Yes it's true. The integration of showing drone zones would be very useful. Maybe it is too far a step with the integration of a commercial drone photo uploading site but I did not find anything like Mapillar for drone photos and it would be very useful when orienting yourself in the field or looking for an object, especially in wild terrain.

**Comment by tradavyvy** [comment]
I have no principal problem with integrating (drone) photos. But the programmers have limited (time) resources and a drone fly-zone-map seems to me more of a priority thing. Besides the photo's would be most usefull in remote terrain where there often is no internet and especialy the offline functions are one of the huge strengths of osmand.

**Comment by vshcherb** [comment]
Is this data present on OpenStreetMap ?

**Comment by patranger** [comment]
As far as I know, this data is not presented anywhere on OpenStreetMap because, as the name suggests, this project has nothing to do with flights and the airspace.

Such data is available in other projects:
https://www.opendronemap.org/
https://www.openflightmaps.org/
